src/main/java/net/datafaker/providers/base/Pronouns.java

```diff
@@ -0,0 +1,44 @@
+package net.datafaker.providers.base;
+
+/**
+ * This class generates gender neutral pronouns
+ * @since 2.1.0
```

Comment:
I believe the next release is slated to be 2.2.0

Reply:
Waiting for the resolution of this comment https://github.com/datafaker-net/datafaker/pull/1106#discussion_r1510218013